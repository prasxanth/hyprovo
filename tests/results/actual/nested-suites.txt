2021-11-24 17:25:52.163 - INFO - Pass ✓... (suite-check): add pass
2021-11-24 17:25:52.165 - ERROR - Fail ✗... (suite-check): sub fail
2021-11-24 17:25:52.167 - ERROR - Fail ✗... (suite-check): div fail
2021-11-24 17:25:52.168 - DEBUG - (suite-check): Test Summary → Total: 3    Pass: 1    Fail: 2
2021-11-24 17:25:52.175 - INFO - Pass ✓... (suite-deftest / test-add): add pass
2021-11-24 17:25:52.177 - ERROR - Fail ✗... (suite-deftest / test-add): add fail
2021-11-24 17:25:52.181 - INFO - Pass ✓... (suite-deftest / test-sub): sub pass
2021-11-24 17:25:52.183 - ERROR - Fail ✗... (suite-deftest / test-sub): sub fail
2021-11-24 17:25:52.183 - DEBUG - (suite-deftest): Test Summary → Total: 4    Pass: 2    Fail: 2
2021-11-24 17:25:52.190 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results): add pass
2021-11-24 17:25:52.192 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results): sub fail
2021-11-24 17:25:52.197 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub pass
2021-11-24 17:25:52.200 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub fail
2021-11-24 17:25:52.204 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div pass
2021-11-24 17:25:52.207 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div fail
2021-11-24 17:25:52.207 - DEBUG - (suite-nested-deftests / test-arithmetic-1-report-combined-results): Test Summary → Total: 6    Pass: 3    Fail: 3
2021-11-24 17:25:52.211 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add pass
2021-11-24 17:25:52.213 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add fail
2021-11-24 17:25:52.218 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul pass
2021-11-24 17:25:52.220 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul fail
2021-11-24 17:25:52.221 - DEBUG - (suite-nested-deftests / test-arithmetic-2-report-combined-results): Test Summary → Total: 4    Pass: 2    Fail: 2
2021-11-24 17:25:52.221 - DEBUG - (suite-nested-deftests): Test Summary → Total: 10    Pass: 5    Fail: 5
2021-11-24 17:25:52.221 - DEBUG - (nested-suites): Test Summary → Total: 17    Pass: 8    Fail: 9
