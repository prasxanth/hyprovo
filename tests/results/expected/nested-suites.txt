2021-11-24 17:24:55.743 - INFO - Pass ✓... (suite-check): add pass
2021-11-24 17:24:55.745 - ERROR - Fail ✗... (suite-check): sub fail
2021-11-24 17:24:55.748 - ERROR - Fail ✗... (suite-check): div fail
2021-11-24 17:24:55.748 - DEBUG - (suite-check): Test Summary → Total: 3    Pass: 1    Fail: 2
2021-11-24 17:24:55.756 - INFO - Pass ✓... (suite-deftest / test-add): add pass
2021-11-24 17:24:55.759 - ERROR - Fail ✗... (suite-deftest / test-add): add fail
2021-11-24 17:24:55.764 - INFO - Pass ✓... (suite-deftest / test-sub): sub pass
2021-11-24 17:24:55.766 - ERROR - Fail ✗... (suite-deftest / test-sub): sub fail
2021-11-24 17:24:55.766 - DEBUG - (suite-deftest): Test Summary → Total: 4    Pass: 2    Fail: 2
2021-11-24 17:24:55.773 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results): add pass
2021-11-24 17:24:55.776 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results): sub fail
2021-11-24 17:24:55.780 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub pass
2021-11-24 17:24:55.783 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-sub): sub fail
2021-11-24 17:24:55.789 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div pass
2021-11-24 17:24:55.792 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-1-report-combined-results / test-div): div fail
2021-11-24 17:24:55.792 - DEBUG - (suite-nested-deftests / test-arithmetic-1-report-combined-results): Test Summary → Total: 6    Pass: 3    Fail: 3
2021-11-24 17:24:55.797 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add pass
2021-11-24 17:24:55.799 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-add): add fail
2021-11-24 17:24:55.805 - INFO - Pass ✓... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul pass
2021-11-24 17:24:55.808 - ERROR - Fail ✗... (suite-nested-deftests / test-arithmetic-2-report-combined-results / test-mul): mul fail
2021-11-24 17:24:55.808 - DEBUG - (suite-nested-deftests / test-arithmetic-2-report-combined-results): Test Summary → Total: 4    Pass: 2    Fail: 2
2021-11-24 17:24:55.808 - DEBUG - (suite-nested-deftests): Test Summary → Total: 10    Pass: 5    Fail: 5
2021-11-24 17:24:55.808 - DEBUG - (nested-suites): Test Summary → Total: 17    Pass: 8    Fail: 9
